mitodl/mit-open
# remove elementary / primary school as an education option

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5211

### Description (What does it do?)
This PR removes elementary / primary school as an option you can select for education level in both onboarding and profile editing in the dashboard

### How can this be tested?
 - Spin up `mit-learn` on this branch
 - Log in with your account
 - Visit http://localhost:8063/onboarding
 - Go through the onboarding steps until you reach the education level question and ensure that the elementary / primary school option is not in the dropdown
 - In the upper right, click your name and then click Dashboard
 - Click on the Profile menu option on the left
 - Verify that elementary / primary school option is gone from the education level dropdown here as well
